python/docs/source/getting_started/quickstart_connect.ipynb

```diff
@@ -0,0 +1,140 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Quickstart: Spark Connect\n",
+    "\n",
+    "This is a short introduction and quickstart for Spark Connect. Spark Connect consists of server and client. This notebook demonstrates a simple step-by-step example of how it works on the server and on the client for new commers to Spark Connect."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Launch Spark Connect server\n",
+    "\n",
+    "Launching server can be done by running `start-connect-server.sh` script as below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!./sbin/start-connect-server.sh --packages org.apache.spark:spark-connect_2.12:3.4.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect from client to server\n",
+    "\n",
+    "Now the Spark Connect server is running, we can connect from client to server. Spark Connect server can be accessed via remote `SparkSession`. Before creating remote `SparkSession`, make sure to stop the existing regular `SparkSession` as below because the regular `SparkSession` and remote `SparkSession` cannot coexist."
```

Comment:
```suggestion
    "Now the Spark Connect server is running, we can connect to the server. Spark Connect server can be accessed via the client, remote `SparkSession` configured for the server. Before creating the remote `SparkSession`, make sure to stop the existing regular `SparkSession` as below because the regular `SparkSession` and remote `SparkSession` cannot coexist."
```